extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -278,15 +295,15 @@ private function sendToPhalanxDaemon( $action, $parameters ) {
 					$res = ( is_null( $response ) ) ? false : $response;
 					break;
 				case "status":
-					$res = ( stripos( $response, self::RES_STATUS  ) !== false ) ? true : false;
+					$res = ( stripos( $response, self::RES_STATUS ) !== false ) ? true : false;
 					break;
 				case "match" :
 					$ret = json_decode( $response );
 					if ( !is_array( $ret ) ) {
 						$res = false;
 					}
 					else {
-						if (count($ret)>0 && $this->limit != 0) {
+						if ( count( $ret ) > 0 && $this->limit != 0 ) {
 							if ( $this->limit == 1 ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)